teracopy-np.json

```diff
@@ -0,0 +1,25 @@
+{
+    "homepage": "https://www.codesector.com",
+    "description": "A popular utility designed to copy files faster and more reliably, providing the user with many features.",
+    "version": "3.26",
+    "url": "https://www.codesector.com/files/teracopy.exe",
+    "hash": "5b667ff25a5f2ecf810e2f60dd079dc68fce6eafa174e6de835b8aed919e12f3",
+    "installer": {
+        "file": "teracopy.exe",
+        "args": [
+            "/verysilent",
+            "/nocancle",
+            "/norestart"
+        ]
+    },
+    "uninstaller": {
+        "script": "& \"$Env:Programfiles\\TeraCopy\\unins000.exe\" /verysilent /nocancle /norestart"
```

Comment:
Shouldn't that be `/nocancel`?

Reply:
oh sorry

Reply:
You haven't fixed this one. Only the first occurrence of `nocancle`.